data/fleet-commander-standalone.in

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function usage {
```

Comment:
Why don't we show exactly the same options that the python module accepts and just pass $@/$* accordingly?

ArgumentParser also does the usage text for you automatically, we should delegate this on the python module